<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>571</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Marcha4Cam - Dos Cámaras Cuerpo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{background-color:white;}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="FeedLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelAnguloRodilla">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{background-color:yellow;}
</string>
         </property>
         <property name="text">
          <string>Ángulo de la rodilla: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PlotWidget" name="graphicsView"/>
       </item>
      </layout>
     </item>
     <item row="0" column="2" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="FeedLabel_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelAnguloCadera">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{background-color:red;}
</string>
         </property>
         <property name="text">
          <string>Ángulo de la cadera: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PlotWidget" name="graphicsCadera"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="CancelBTN">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background-color:skyblue;}
QPushButton:hover{background-color:orange;}</string>
       </property>
       <property name="text">
        <string>Cerrar</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QPushButton" name="btnStart">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background-color:skyblue;}
QPushButton:hover{background-color:orange;}</string>
       </property>
       <property name="text">
        <string>Comenzar análisis</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btnStop">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background-color:skyblue;}
QPushButton:hover{background-color:orange;}
</string>
       </property>
       <property name="text">
        <string>Detener análisis</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
